test/verify/check-pages

```diff
@@ -396,6 +396,26 @@ OnCalendar=daily
 
         self.allow_restart_journal_messages()
 
+    def testShellPreload(self):
```

Comment:
This neeeds to be skipped on [rhel-8-1-distropkg](https://209.132.184.41:8493/logs/pull-12534-20190814-064040-567c60d4-cockpit-project-cockpit-rhel-8-0-distropkg/log.html#239)

Reply:
Done.